client/input.go

```diff
@@ -28,12 +28,15 @@ func GetPassword(prompt string, buf *bufio.Reader) (pass string, err error) {
 	} else {
 		pass, err = readLineFromBuf(buf)
 	}
+
 	if err != nil {
 		return "", err
 	}
+
 	if len(pass) < MinPassLength {
-		return "", errors.Errorf("password must be at least %d characters", MinPassLength)
+		return pass, errors.Errorf("password must be at least %d characters", MinPassLength)
```

Comment:
Why was this changed? We shouldn't return anything here if it errored.

Reply:
I changed it because otherwise it won't work when you want to "default" aka, `pass = ""`. Alternately, we can create a custom error and do: `if err == customError && pass == ""`

Reply:
Alternatively, we can remove the notion of a default key password all together making this super simple.

Reply:
Ohh I see. Then what you wrote is fine, lets just add a comment. 

> Alternatively, we can remove the notion of a default key password all together making this super simple.

I'm fine with that as well, but I think that should be discussed in an issue first, since thats a UI change and I'd prefer feedback from UX/Design (or at least more people) about good flows for this stuff.

Reply:
Yup. Agreed. I'll comment.